libr/anal/anal.c

```diff
@@ -453,7 +454,10 @@ static int nonreturn_print(void *p, const char *k, const char *v) {
 		}
 	}
 	if (!strncmp (k, "addr.", 5)) {
-		char *off = strdup (k + 5);
+		char *off;
+		if (!(off = strdup (k + 5))) {
+			return 1;
```

Comment:
maybe need to do something else instead ``return 1;`` in this case?

Reply:
leaking op

Reply:
Where? I don't see op variable in this scope